### DISCUSSION TITLE
Sharing my original Airton module

### DISCUSSION THREAD

**Discussion by Neuvidor** [ouinouin/Berryton#7] [General]
Hi,
I have a second original Airton Wifi module (not used, as spare part) that I can send you for free (only shipping) if it can help you during the development time. I let you free to contact me if you are interrested. I'm in the north east or France. Best regards.

**Comment by maxencel-dev** [comment]
Salut,j'ai 3 clim de marque SnowAir que j'ai acheté en Espagne il y a quelques années. Je pense que c'est une clone de l'Airton.
Serait-il possible d'avoir ton module en prêt afin de voir s'il est compatible?
Bàt
